src/main/java/com/radixdlt/crypto/ECDSASignature.java

```diff
@@ -126,4 +131,20 @@ public int hashCode() {
 	public SignatureScheme signatureScheme() {
 		return SignatureScheme.ECDSA;
 	}
+
+	public static ECDSASignature decodeFromDER(byte[] bytes) {
+		DLSequence seq;

```

Comment:
Since only the happy path returns anything, these variables and the return statement can be compressed into the `try` block

Reply:
nice! good improvement!